lib/hooks.ts

```diff
@@ -34,42 +35,48 @@ export interface UsersData {
 export interface UseAppDataStore {
   data?: UsersData
   error?: unknown
-  selectedIndex: number | undefined
+  selectedUser?: UserRow
+  attachments: File[]
+  warnings: PreviewWarning[]
   setData: (data: UsersData | undefined) => void
+  addAttachments: (files: File[]) => void
+  removeAttachment: (file: File) => void
 }
 
-export const useAppDataStore = create<UseAppDataStore>((set) => ({
+export const useAppDataStore = create<UseAppDataStore>((set, get) => ({
   data: undefined,
-  selectedIndex: undefined,
   error: undefined,
+  selectedUser: undefined,
+  attachments: [],
+  warnings: [],
```

Comment:
Why did you move the attachments here from `useTemplateStore`?  I feel like the
template store is a more logical place than the app store but maybe I'm missing something.

Reply:
> Why did you move the attachments here from `useTemplateStore`? I feel like the template store is a more logical place than the app store but maybe I'm missing something.

I think putting warnings, attachments inside `useAppDataStore` makes more sense